hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/DeleteMetadataActor.scala

```diff
@@ -54,7 +54,8 @@ class DeleteMetadataActor(metadataDeletionConfig: ActiveMetadataDeletionConfig,
             numOfWorkflowsToDeleteMetadataMetricActor ! CalculateMetricValue(metricCalculatingFunction)
           }
           workflowIds foreach { workflowIdStr =>
-            deleteNonLabelMetadataEntriesForWorkflowAndUpdateArchiveStatus(WorkflowId.fromString(workflowIdStr), MetadataArchiveStatus.toDatabaseValue(ArchivedAndPurged)) onComplete {
+            // TODO: This is no longer working as intended! We should delete rather than patch it.
+            deleteAllMetadataEntriesForWorkflowAndUpdateArchiveStatus(WorkflowId.fromString(workflowIdStr), MetadataArchiveStatus.toDatabaseValue(ArchivedAndPurged)) onComplete {
```

Comment:
Pretty sure this will have been deleted on `develop` thanks to https://github.com/broadinstitute/cromwell/pull/6234 so no worries on leaving it in